tests/func/experiments/test_remove.py

```diff
@@ -179,3 +179,162 @@ def test_remove_multi_rev(tmp_dir, scm, dvc, exp_stage):
 
     assert scm.get_ref(str(baseline_exp_ref)) is None
     assert scm.get_ref(str(new_exp_ref)) is None
+
+
+def test_keep_selected_by_name(tmp_dir, scm, dvc, exp_stage):
+    # Setup: Run experiments
+    results = dvc.experiments.run(exp_stage.addressing, params=["foo=1"], name="exp1")
+    exp1_ref = first(exp_refs_by_rev(scm, first(results)))
+
+    results = dvc.experiments.run(exp_stage.addressing, params=["foo=2"], name="exp2")
+    exp2_ref = first(exp_refs_by_rev(scm, first(results)))
+
+    results = dvc.experiments.run(exp_stage.addressing, params=["foo=3"], name="exp3")
+    exp3_ref = first(exp_refs_by_rev(scm, first(results)))
+
+    # Ensure experiments exist
+    assert scm.get_ref(str(exp1_ref)) is not None
+    assert scm.get_ref(str(exp2_ref)) is not None
+    assert scm.get_ref(str(exp3_ref)) is not None
+
+    # Keep "exp2" and remove others
+    removed = dvc.experiments.remove(exp_names=["exp2"], keep=True)
+    assert sorted(removed) == ["exp1", "exp3"]
+
+    # Check remaining experiments
+    assert scm.get_ref(str(exp1_ref)) is None
+    assert scm.get_ref(str(exp2_ref)) is not None
+    assert scm.get_ref(str(exp3_ref)) is None
+
+
+def test_keep_selected_multiple_by_name(tmp_dir, scm, dvc, exp_stage):
```

Comment:
let's merge this test with the previous one and parametrize pls (pass one name to remove, two names, three names)

Reply:
let me know if you need some example on how to parametrize pytest

Reply:
I tried by myself before reaching out for help, and it seems I managed to get something nice working with this parametrization :-) Thanks for the idea ! I'll reuse that for sure.